### PR TITLE
Add documentation for the Flo integration

### PR DESCRIPTION
**Pull request by dmulcahey**
## Proposed change

This PR adds initial documentation for the Flo integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/38171
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/flo_moen.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Flo by Moen
+description: Instructions on how to integrate Flo by Moen into Home Assistant.
+ha_release: '0.115'
```
Comment (klaasnicolaas) [inline source/_integrations/flo_moen.markdown]:
```suggestion
ha_release: 0.115
```

File: source/_integrations/flo_moen.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+[Flo by Moen smart water shutoff valves](https://meetflo.com/product/smart-water-shutoff) into Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- **Sensor**: reports on the device's system mode, water flow rate, temperature, water pressure, and daily water consumption.
+
+## Configuration
+
+This integration can be configured via the Home Assistant UI by navigating to
+**Configuration** -> **Integrations**.
```
Comment (klaasnicolaas) [inline source/_integrations/flo_moen.markdown]: And click on the `+` sign and search for `flo`...?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
I think it's better to rename the file to `flo.markdown`

File: source/_integrations/flo_moen.markdown (hunk 1 above)
Comment (dmulcahey) [reply source/_integrations/flo_moen.markdown]: done

File: source/_integrations/flo_moen.markdown (hunk 2 above)
Comment (dmulcahey) [reply source/_integrations/flo_moen.markdown]: addressed

**Review by frenck — APPROVED** [review]
Thanks, @dmulcahey! 👍 

Can we add a PR with the icons/logos to the brands repository as well? 😃

**Comment by dmulcahey** [comment]
@frenck https://github.com/home-assistant/brands/pull/1793

**Pull request merged**